dl1_data_handler/reader.py

```diff
@@ -306,6 +305,12 @@ def __init__(self,
                  transforms=None,
                  validate_processor=False
                 ):
+
+        # Import ctapipe DL1 reader
+        try:
+            from ctapipe.io import read_table # let us read full tables inside the DL1 output file
```

Comment:
You require `ctapipe >= 0.10.5` in the setup.py. That provides read_table. Why is this try except needed?

Reply:
Hi @maxnoe. Thanks for the catch! True, it isn't needed.